src/main/java/org/folio/services/transactions/TransactionRestrictService.java

```diff
@@ -48,15 +48,15 @@ private Future<Void> checkFundsAllocatedIds(Transaction transaction, RequestCont
         .map(cf -> isAllocationAllowed(fromFund.result(), toFund.result(), transaction))
         .compose(isAllocationAllowed -> {
           if (Boolean.TRUE.equals(isAllocationAllowed)) {
-            log.info("checkFundsAllocatedIds: Allocation is allowed");
+            log.info("checkFundsAllocatedIds: Allocation is allowed for transaction: {}", transaction);
```

Comment:
the same transactionId or it has toString overriden?

Reply:
as you said there should be transactionid